### ISSUE TITLE
Hotkeys are ignored after wake-up from sleep

### ISSUE THREAD

**Issue by desmap** [xanderfrangos/twinkle-tray#132]
Pretty much the title. If I left-click on the twinkie tray's tray icon, hotkeys work again.

**Comment by xanderfrangos** [comment]
I'm having some trouble recreating the issue on my end. Are you running the latest version of Twinkle Tray (v1.13.3)?

**Comment by desmap** [comment]
Yes
![image](https://user-images.githubusercontent.com/43666255/108406890-4ad7a380-7223-11eb-9aa2-de340739ac9f.png)

This started just recently, IDK maybe a few days ago, before everything was fine. It's also not always after wake-up but sometimes. I can observe it and maybe I find more. Btw, awesome tool you wrote, thanks!

**Comment by desmap** [comment]
Not sure if this is related: sometimes when the hotkeys are working the visual indication at the top left of the screen isn't and I get something like this (it just happened after it was working fine):
![image](https://user-images.githubusercontent.com/43666255/108407908-a8b8bb00-7224-11eb-8b82-920c59517094.png)

**Comment by xanderfrangos** [comment]
Yeah, it's probably related (and good to know). Since it only broke recently, I at least have an idea of where the issue could be.

Could you copy/paste your monitor info and settings from the Debug menu (bottom left of the Settings window)? Also, if you click the "Refresh Monitors" button (in the Debug menu) a bunch of times, do any of your monitors disappear or change?

**Comment by desmap** [comment]
here you go with the info and pls fid below more re the refresh button:
```
Name: Acer PE270K
Internal name: ACR0621
Communication Method: DDC/CI 
Current Brightness: 41
Max Brightness: 100
Raw Brightness: 41
Features: {"luminance":[55,100],"brightness":false,"gain":false,"contrast":false,"powerState":false,"volume":false}
Brightness Normalization: 0 - 100
Order: 0
Key: REDACTED
ID: REDACTED
Connection Type: displayport_external

Settings
These are your raw user settings.
{
  "isDev": false,
  "userClosedIntro": true,
  "theme": "default",
  "icon": "mdl2",
  "updateInterval": 250,
  "openAtLogin": true,
  "brightnessAtStartup": true,
  "killWhenIdle": false,
  "remaps": {
    "REDACTED": {
      "min": 0,
      "max": 100
    }
  },
  "hotkeys": {
    "all__dir-1": {
      "monitor": "all",
      "accelerator": "F7",
      "direction": -1,
      "active": true
    },
    "all__dir1": {
      "monitor": "all",
      "accelerator": "F8",
      "direction": 1,
      "active": true
    }
  },
  "hotkeyPercent": 10,
  "adjustmentTimes": [],
  "adjustmentTimeIndividualDisplays": false,
  "checkTimeAtStartup": true,
  "order": [],
  "features": {
    "luminance": true,
    "brightness": true,
    "gain": false,
    "contrast": false,
    "powerState": false,
    "volume": false
  },
  "checkForUpdates": true,
  "dismissedUpdate": "",
  "language": "system",
  "settingsVer": "v1.13.3",
  "names": {},
  "analytics": false,
  "scrollShortcut": true,
  "useAcrylic": true,
  "useNativeAnimation": false,
  "uuid": "REDACTED",
  "branch": "master"
}
```
> Also, if you click the "Refresh Monitors" button (in the Debug menu) a bunch of times, do any of your monitors disappear or change?

Nothing happens but this thing doesn't feel like a button. When I've clicked to often it select the button text.

**Comment by xanderfrangos** [comment]
😅 Yeah, the refresh button wasn't meant for regular users. If you didn't see any changes after clicking it, that's fine.

Thanks for the info, this helps. If you notice anything else, let me know.

**Comment by desmap** [comment]
Just found one more hint:
- If my monitor goes into standby (after the notebook went to sleep) and wakes up again (after the notebook woke up) all good
- But if I (sometimes I do) turn off explicitly the monitor after the notebook went to sleep and then turn it on again after the notebook woke up then the hotkeys are ignored but the visual indication is still showed and accordingly changes to the keystrokes buy the actual brightness stays the same.

**Comment by desmap** [comment]
FWIW, what I wrote before is not true—the issue also happens after the monitor wakes up from standby (the case from the first bullet).

**Comment by gggomg** [comment]
@xanderfrangos Good day! I'facing same issue, as solution maybe you can add hotkey setting for Update monitors button in Tray? Because only this button helps with this problem after display is awakened.

**Comment by gggomg** [comment]
@sovon @desmap ok after some experiments these are settings have worked for me and now twinkle works correctly after waking up. Ty
![image](https://github.com/user-attachments/assets/061c75b0-e565-46e2-8081-b6abc066b97d)
